docs/tutorial.md

```diff
@@ -0,0 +1,372 @@
+---
+title: Quarkus Operator Tutorial
+linkTitle: Tutorial
+weight: 30
+description: An in-depth walkthough of building and running a Quarkus-based operator.
+---
+
+**NOTE:** If your project was created with an `operator-sdk` version prior to `v1.0.0`
+please [migrate][migration-guide], or consult the [legacy docs][legacy-quickstart-doc].
+
+## Prerequisites
+
+- Go through the [installation guide][install-guide].
```

Comment:
broken link where is the url for the install guide?

Reply:
Modified with Java link.